README.md

```diff
@@ -5,6 +5,7 @@ curl-to-Go is a tool to instantly convert [curl](http://curl.haxx.se) commands t
 
 This is a sister tool to [JSON-to-Go](https://mholt.github.io/json-to-go), which converts JSON to a Go struct type definition.
 
+To do the reverse, check out [moul/http2curl](https://github.com/moul/http2curl).
```

Comment:
To do the reverse of JSON-to-Go-struct conversion? ;) It's a little confusing here, let's move this up to the end of the first paragraph. Thanks!